plugins/git/git.plugin.zsh

```diff
@@ -255,3 +255,21 @@ alias glum='git pull upstream master'
 
 alias gwch='git whatchanged -p --abbrev-commit --pretty=medium'
 alias gwip='git add -A; git rm $(git ls-files --deleted) 2> /dev/null; git commit --no-verify --no-gpg-sign -m "--wip-- [skip ci]"'
+
+function grename() {
+  if [ -z "$1" ]
+  then
+    echo "use command like 'grename old_branch new_branch'"
+    return
+  fi
+
+  if [ -z "$2" ]
+  then
+    echo "use command like 'grename old_branch new_branch'"
+    return
+  fi
```

Comment:
This check can be added to the previous if

Reply:
I have added next if on a single line